### PR TITLE
Changes to allow to locate the app and Translation to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by tiagocanto01**

### CODE REVIEW

**Comment by zackshapiro** [comment]
How does one set their language in this?

**Comment by tiagocanto01** [comment]
To add a new language, you could select the Raiblocks Project -> Info -> Localizations. There, you can see all the languages ​​that the application currently supports. To add a new language, you could click the + button, choose the language and select only the Localizable.strings file.

In the file tree, you can see the Localizable.strings in the Raiblocks folder. If you expand you can see the languages, there is a file for each language. Choose the language you want to translate. The strings on the left side should be in English, the strings on the right side should be the translations.

If a new text is inserted into the application, it is necessary to call the localized() function ("<text>".localized()) and add this text in all the files to be translated. If you don't add, the application uses the English text, by default.
If you change some text in English the application can't find the translation and will be shown the text in English too, until someone changes int the localizable file with the translation.

PS: I am trying to improve my communication in English, if anything is not clear, you can ask again and I will try to explain in other ways.

captura de tela 2018-06-26 as 13 21 54

**Comment by zackshapiro** [comment]
I was more asking about the user's experience in the app, how they select from English or Brazilian Portuguese 

Keep practicing your English, glad to have you here :)

**Comment by tiagocanto01** [comment]
It's automatic, if the iPhone is set to Portuguese, the app will show in Portuguese, otherwise show in english.

File: Raiblocks/Extensions/String+Extensions.swift [hunk 1]
```diff
@@ -34,4 +34,7 @@ extension String {
         }
     }
 
+    func localized(comment: String = "") -> String {
+        return NSLocalizedString(self, comment: comment)
```
Comment (zackshapiro) [inline Raiblocks/Extensions/String+Extensions.swift]: Is this what's looking at the system language setting? How is Localizable.strings being used?

Comment (tiagocanto01) [reply Raiblocks/Extensions/String+Extensions.swift]: Exactly. You can read this article, which explains better how it works: https://medium.com/lean-localization/ios-localization-tutorial-938231f9f881

**Comment by zhyatt** [comment]
The NanoWalletCompany is no longer supporting this app. See https://nanowalletcompany.com/ for details. Closing.

**Pull request closed without merging**